feebleline.el

```diff
@@ -214,6 +216,25 @@ Returns a pair with desired column and string."
   (with-current-buffer feebleline--minibuf
     (erase-buffer)))
 
+
+(defun feebleline--count-windows ()
+  "Total window count."
+  (let ((count 0))
+    (dolist (fn (frame-list))
+      (setq count (+ (length (window-list fn)) count)))
+    count))
+
+(defun feebleline--walk-through-windows (fnc)
+  "Walk through all the windows once and execute callback FNC."
+  (save-selected-window
+    (let ((index 0))
+      (while (< index (feebleline--count-windows))
+        (when fnc
+          (funcall fnc))
+        (other-window 1 t)
+        (setq index (+ index 1))))))
```

Comment:
This function just need a function pointer `fnc`, and call it every time when the window changed.